src/pythoshop/resize_image.py

```diff
@@ -0,0 +1,76 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+
+def resize_image(image_path: str, height: int, width: int, method: str = 'crop', verbose: bool = True):
+    """
+    Image Resizing Function
+
+    This function takes an image path and adjusts the image to have
+    the inputted dimensions using the selected method.
+
+    Parameters:
+    - image_path (str): The path to the input image.
+    - height (int): The desired height for the image.
+    - width (int): The desired width for the image.
+    - method (str, optional): The method to obtain the desired image dimensions.
+      Options: 'maintain_aspect_ratio' (default) for maintaining the aspect ratio,
+               'crop' for cropping to the specified dimensions,
+               'add_borders' for adding borders to maintain the aspect ratio.
+    - verbose (bool, optional): If True, print verbose information.
+
+    Returns:
+    - None
+    The resized image is saved as a .png file at the same location as the input with "_res_img.png" appended
+    to the original filename.
+
+    Raises:
+    - IOError: If the image file cannot be opened or saved.
```

Comment:
I think you can delete this line about raising an IOError since you raise a FileNotFoundError in your function instead.